src/coreclr/src/gc/gc.cpp

```diff
@@ -15583,6 +15591,7 @@ int gc_heap::joined_generation_to_condemn (BOOL should_evaluate_elevation,
             // where foreground GCs are asking for a compacting full GC right away
             // and not getting it.
             dprintf (GTC_LOG, ("full GC induced, not reducing gen"));
+            gc_data_global.gen_to_condemn_reasons.set_condition(gen_joined_user_requested);
```

Comment:
>gen_joined_user_requested [](start = 64, length = 25)

I would call this gen_joined_pm_induced_fullgc_p to be consistent with the per heap one which is gen_induced_fullgc_p, and also emphasizing the PM part